From ce4911c13a3ab6877c0288c8a1874decb5d0e90a Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Mon, 23 Jul 2012 09:14:24 -0300
Subject: [PATCH] Makefile: fix high jobcount build failures

gram.h is a dependency for scanner.c rather than scanner.o which is
unused.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 04b834b..8a90ca7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -131,5 +131,5 @@ dist-hook:
 	rm -f $(distdir)/gram.h
 	rm -f $(distdir)/scanner.c
 
-scanner.o: gram.h
+scanner.c: gram.h
 
-- 
1.7.8.6

